src/strategies/spidStrategy.ts

```diff
@@ -129,4 +134,25 @@ const spidStrategy = async (
   );
 };
 
+/**
+ * Reads dates information in x509 certificate and logs remaining time to its expiration date.
+ * @param samlCert x509 certificate as string
+ */
+function logSamlCertExpiration(samlCert: string): void {
+  try {
+    const out = x509.parseCert(samlCert);
+    if (out.notAfter) {
+      const timeDiff = distanceInWordsToNow(out.notAfter);
```

Comment:
did you check what comes out for future dates? (since this is distance to now, not from now)

Reply:
Distance string values are always in absolute value. Fixed for notAfter date in the past in commit 09cf6bf